package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "eslint": "eslint src",
     "eslint-fix": "npm run eslint -- --fix",
-    "prepare": "rollup -c",
+    "preparea": "rollup -c",
```

Comment:
Was this intended? I indeed have no idea about what it should do, but it seems a lot like a typo to me 😀

Reply:
Haha, it was a temporary rename because `npm install` was failing 🙃

Reply:
So yes, typo. You can push up a fix.